EventFilter/SiPixelRawToDigi/plugins/RawToDigiGPU.h

```diff
@@ -145,6 +146,12 @@ inline uint32_t pack(uint32_t row, uint32_t col, uint32_t adc) {
 
 }
 
+typedef struct error {
+    uint32_t rawId;
+    uint32_t word;
+    unsigned char errorType;
+    unsigned char fedId;
+    } error_obj;
```

Comment:
Is there any reason to not be fully C++ here (i.e. drop typedef and the trailing name)?

Reply:
+1